app/styles/front.scss

```diff
@@ -708,33 +686,60 @@ legend {
   width: 1px;
 }
 
-input:checked + .radioButton, input:checked + .radioButton:active:hover {
+.inputButton {
```

Comment:
Wouldn't that be too general ? What if somewhere there is a checkbox before a label where we don't want to apply this design ?

For instance the "Je continuerai à percevoir..."